backend/package.json

```diff
@@ -51,6 +51,7 @@
     "helmet": "5.1.1",
     "inversify": "6.0.1",
     "inversify-express-utils": "6.4.3",
+    "jose": "^4.8.3",
```

Comment:
Check quality criteria. You cannot use `^`

Rule B19

The dependency lists in the package.json file indicate the exact versions of the packages used. The version must be specified. ^, * and ~ are not allowed.